java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_2/BirthDayValidator.java

```diff
@@ -0,0 +1,24 @@
+package com.zoolatech.lecture3.tasks._2;
+
+public class BirthDayValidator implements ValidatorInterface {
```

Comment:
You didn't need to create a validator class for each field, you needed to create a validator class for each rule:

```
 List of validation rules: string is not empty, valid email, valid phone number, number is in the range between X and Y, number is bigger than a value X, number is less than a value X. 
 ```